LiskHQ/lisk-sdk
# Fix calculation of listOfCCUs on cleanup

### What was the problem?

This PR resolves #8321 

### How was it solved?

Taking correct range of CCUs to be saved : [🐛 Update listOfCCUs bug on cleanup](https://github.com/LiskHQ/lisk-sdk/commit/d823f43e92ead15ff342c24183efe1efc1040799)

### How was it tested?

- Setup example interop applications, do the registration and watch CCU transactions.
- Set `ccuSaveLimit` as 2 and check if the last 2 CCUs are saved using `chainConnector_getSentCCUs` endpoint
